nodes/standalone/src/service.rs

```diff
@@ -52,11 +52,15 @@ pub(crate) type FullClient = sc_service::TFullClient<Block, RuntimeApi, NativeEl
 type FullBackend = sc_service::TFullBackend<Block>;
 type FullSelectChain = sc_consensus::LongestChain<FullBackend, Block>;
 
+/// The minimum period of blocks on which justifications will be
+/// imported and generated.
+const GRANDPA_JUSTIFICATION_PERIOD: u32 = 512;
```

Comment:
Where was this value taken from? Can you put a reference in a comment here?

Reply:
Like so: [ba967f1](https://github.com/KILTprotocol/kilt-node/pull/627/commits/ba967f1d7ba79a529b359a3d7f94636caa044c0b)?